### PR TITLE
new display changes

### PR DESCRIPTION
**Pull request by ssozi91**

### CODE REVIEW

File: routes/auth-routes.js [hunk 1]
```diff
@@ -52,7 +46,7 @@ router.post('/verif', function (req, res) {
         //updating the user table with user verification code
         else {
           if (!user.rowCount) {
-            res.redirect('/auth/verifAgain');
+            res.redirect('http://localhost:3000/Activation');
```
Comment (laurieainley) [inline routes/auth-routes.js:49]: Why have you added the domain to this URL (http://localhost:3000)? Once this moves to its own domain e.g. http://convenient.com, would this cause a problem?

File: routes/auth-routes.js [hunk 2]
```diff
@@ -71,7 +65,7 @@ router.post('/verif', function (req, res) {
               return console.log('am the ', error);
             }
             else {
-              res.redirect('http://localhost:3000/dashboard');
+              res.redirect('http://localhost:3000/Displayslots');
```
Comment (laurieainley) [inline routes/auth-routes.js:66]: Same as line 49 and line 100

File: src/container/Displayslots.js [hunk 3]
```diff
@@ -1,13 +1,16 @@
...
 import "./Home.css";
-// import Button from "../components/Button.js";
 import Header from "../components/Header.js";
 import Footer from "../components/Footer.js";
 import { NavLink} from 'react-router-dom';
 import Image from "../components/Image.js";
-import { Container, Card, CardTitle, CardText, Row, Col, Button } from 'reactstrap';
-import moment from 'moment';
+import moment from 'moment'
+import { Dropdown, DropdownToggle, DropdownMenu, DropdownItem, Button } from 'reactstrap';
```
Comment (laurieainley) [inline src/container/Displayslots.js:9]: Line 3 is importing from reactstrap too, should these be combined for clarity?

**Review by laurieainley — COMMENTED** [review]
Looks mostly good

File: src/container/Displayslots.js (hunk 3 above)
Comment (ssozi91) [reply src/container/Displayslots.js:9]: @laurieainley okay i will combine them

**Pull request closed without merging**